src/Infrastructure/LeanCode.ExternalIdentityProviders/Facebook/FacebookClient.cs

```diff
@@ -120,6 +121,20 @@ private FacebookUser ConvertResponse(JsonElement result)
                     return null;
                 }
             }
+
+            static bool GetIsSilhouette(JsonElement element)
+            {
+                if (element.TryGetProperty("picture", out var pic) &&
+                    pic.TryGetProperty("data", out var data) &&
```

Comment:
Unused? Though I'm not sure if there is any reason to check for `picture` and `data` unless it's possible to receive a response that doesn't have them but has `"is_silhouette": true`. 
```suggestion
                if (element.TryGetProperty("picture", out _) &&
                    pic.TryGetProperty("data", out _) &&
```

Reply:
Read closely :P

That's the response:
```
{
  "id": "4922594101147747",
  "name": "Jakub Fijałkowski",
  "picture": {
    "data": {
      "height": 50,
      "is_silhouette": true,
      "url": "https://scontent-waw1-1.xx.fbcdn.net/v/t1.30497-1/cp0/c15.0.50.50a/p50x50/84628273_176159830277856_972693363922829312_n.jpg?_nc_cat=1&ccb=3&_nc_sid=12b3be&_nc_ohc=-8z4hHC_XtoAX_HAa9v&_nc_ht=scontent-waw1-1.xx&tp=27&oh=c69e9a83d261021e2a01a272bb50f618&oe=6062A8B8",
      "width": 50
    }
  }
}
```

Reply:
Ah, damn, I'm blind :stuck_out_tongue: